api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -51,11 +53,27 @@ message IPTagging {
     repeated config.core.v3.CidrRange ip_list = 2;
   }
 
+  // Specifies the content of the IP tag file
+  //
+  // Allow the file to be created with no IP tags.
+  message IPTagFile {
+    repeated IPTag ip_tags = 1;
+  }
```

Comment:
Do we really need to expose this?

Reply:
This makes it easier. I use this in my `TagSetWatcher::protoFromFileContents_` to validate my yaml/json file content and convert it to proto which is later converted into LcTrie.